### PR TITLE
feat: allow trigger of sync on mgmt-ui

### PR DESCRIPTION
**Pull request by albertyfwu**
no that pretty but works for now.

later, we probably want to show this on the Sync Runs page also.
Also, we shouldn't show the button if a sync is already running, for better UX.

image

## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

**Review by lucasmarshall — APPROVED** [review]

File: apps/mgmt-ui/src/components/logs/SyncsTable.tsx [hunk 1]
```diff
@@ -99,6 +100,36 @@ export default function SyncsTable(props: SyncsTableProps) {
...
+    {
+      field: '_',
+      headerName: 'Actions',
+      width: 120,
+      sortable: false,
+      renderCell: (params) => {
+        // button to trigger sync run
+        return (
+          <Button
+            variant="contained"
```
Comment (tomkit) [inline apps/mgmt-ui/src/components/logs/SyncsTable.tsx:112]: nit: `size="small"`

Comment (tomkit) [reply apps/mgmt-ui/src/components/logs/SyncsTable.tsx:112]: nit: https://mui.com/material-ui/react-tooltip/ to help message to the user the impact of a full refresh

**Review by tomkit — APPROVED** [review]
ty

File: apps/mgmt-ui/src/components/logs/SyncsTable.tsx (hunk 1 above)
Comment (albertyfwu) [reply apps/mgmt-ui/src/components/logs/SyncsTable.tsx:112]: there's no full refresh here. we can do that later.

Comment (tomkit) [reply apps/mgmt-ui/src/components/logs/SyncsTable.tsx:112]: sg

thoughts on quick-fix? would adding two buttons to the same row allow the user to do either): `trigger` and `trigger full`?

Comment (albertyfwu) [reply apps/mgmt-ui/src/components/logs/SyncsTable.tsx:112]: ok

**Pull request merged**